docs/UserGuide.md

```diff
@@ -15,32 +15,34 @@ If you can type fast, TA<sup>2</sup> can get your contact and assignment managem
 
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have `Java 11` or later installed in your Computer.
 
-2. Download the latest `ta2.jar` from [here](https://github.com/AY2122S1-CS2103T-T13-2/tp/releases/tag/v1.4).
+2. Download the latest `ta2.jar` release from [here](https://github.com/AY2122S1-CS2103T-T13-2/tp/releases/tag/v1.4).
 
 3. Copy the file to the folder you want to use as the _home folder_ for your TA<sup>2</sup>.
 
-4. Double-click the file to start the app. The GUI similar to the one shown below should appear in a few seconds.
-   Note how the app contains some sample data.<br>
+4. Double-click the file to start the app. The window similar to the one shown below should appear in a few seconds.
```

Comment:
```when you run the...``` is redundant since this instruction is for people who just downloaded the jar file?

Reply:
i agree